src/Lykke.Logs/Slack/LykkeLogToSlack.cs

```diff
@@ -154,13 +178,29 @@ private string GetComponentNamePrefix()
         private string GetComponentName(string component)
         {
             if (AppEnvironment.Name == null || !AppEnvironment.Name.StartsWith(component))
-            {
                 return string.Concat(_componentNamePrefix, " : ", component);
+            return _componentNamePrefix;
+        }
+
+        private bool IsSameMessage(LogLevel level, string message)
+        {
```

Comment:
Is it possible to share the code with this logic and do not produce copy&past?

Reply:
I think filtering functionallity should be delegated to MessagesFilter class